src/Frontend/component/About/AboutUs.jsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
-// import AboutUs1 from './Images/AboutUs.jpeg';
+import './AboutUs.css';
 
 export default function AboutUs() {
   return [
```

Comment:
If you only have one main parent component, It is not really necessary to use square brackets, the best practice is to use just parenthesis and if you don't need a parent component you can use [] :)